assets/app/views/_overview-deployment.html

```diff
@@ -42,11 +42,11 @@ <h3 class="h4">
       </h3>
     </div>
 
-    <div class="component meta-data">
+    <div class="component meta-data" ng-init="causes = (rc | deploymentCauses)">
```

Comment:
I thought this was cleaner because we're not changing the data from the server, but I can go either way.